src/pages/patientView/patientHeader/PatientHeader.tsx

```diff
@@ -10,7 +10,8 @@ import {
 } from 'cbioportal-frontend-commons';
 import SampleManager from '../SampleManager';
 
-import styles from './styles.module.scss';
+// TODO styles.module.scss is empty!
+// import styles from './styles.module.scss';
```

Comment:
remove comments?

Reply:
yeah, eventually we should either remove those invalid usage or update them